scripts/results/session-segments.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python
+#.....................................................................
+# Copyright (c) 2016-2020, Regents of the University of Arizona.
+# Author: Wenkai Zheng <wenkaizheng@email.arizona.edu>
+#         Chavoosh Ghasemi <chghasemi@cs.arizona.edu>
+
+# You should have received a copy of the GNU General Public License along with
+# this script e.g., in COPYING.md file. If not, see <http://www.gnu.org/licenses/>.
+#
+#
+# DESCRIPTION:
+# Feed this file with the collected log file of the consumers,
+# on the server side. You can find this file on the machine
+# that runs the stats-collector tool.
+#
+# The output is a plot, showing the number received Data
```

Comment:
Change it to this:
The output is a plot, showing the number received Data packets by
the consumer during *each* video playing back (i.e, each session).

Reply:
You can barely with the line adjustment you use. Almost always the lines are at equal length or the later ones have a shorter length.

Again, I encourage you to read more and spend more time on repo's source code to prevent such mistakes.

Reply:
I will pay more attention to format problems before I push any pull request.